### PR TITLE
New Relic Documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Migrated to the new doc structure (rebasing was causing some issues)
> Incorporated all the suggestions by @ItsSudip listed [here](https://github.com/rudderlabs/rudderstack-docs/pull/577#pullrequestreview-970804069)

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

## Notion ticket link

> [New Relic documentation](https://www.notion.so/rudderstacks/New-Relic-destination-8268ba8babc044cc98628a8fcb19caa8)

### CODE REVIEW

**Review by rohithkaza — APPROVED** [review]
LGTM

**Review by ItsSudip — APPROVED** [review]

File: docs/destinations/analytics/new-relic.mdx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+<div class="infoBlock">
+    
+To know more about the difference between cloud mode and device mode in RudderStack, refer to the <a href="https://rudderstack.com/docs/rudderstack-cloud/rudderstack-connection-modes/">RudderStack Connection Modes</a> guide.
+</div>
+
+Once you have confirmed that the source platform supports sending events to New Relic, follow these steps:
+
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **New Relic**.
+2. Assign a name to the destination and click on **Next**.
```
Comment (ishachopra0401) [inline docs/destinations/analytics/new-relic.mdx:29]:
```suggestion
2. Assign a name to the destination and click on **Continue**.
```

File: docs/destinations/analytics/new-relic.mdx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+</div>
+
+- **Insert Key**: Enter your New Relic insert key.
+
+<div class="infoBlock">
+
+For more information on getting your New Relic insert key, refer to the <a href="#faq">FAQ</a> section below.
+</div>
+
+- **Data Center**: Choose your data center associated with your New Relic account. By default, it is set to **US**.
```
Comment (ishachopra0401) [inline docs/destinations/analytics/new-relic.mdx:51]:
```suggestion
- **Data Center**: Choose the data center associated with your New Relic account. By default, it is set to **US**.
```

File: docs/destinations/analytics/new-relic.mdx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      }
+    }
+  }
+);
+```
+
+RudderStack applies the following rules to the event properties before sending them to New Relic:
+
+- Booleans are transformed into strings.
+- NRQL words are wrapped with backticks. [Reference](https://docs.newrelic.com/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#:~:text=to%20the%20logs.-,NRQL%20syntax%20terms,-If%20you%20need).
```
Comment (ishachopra0401) [inline docs/destinations/analytics/new-relic.mdx:89]:
```suggestion
- [NRQL words](https://docs.newrelic.com/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#:~:text=to%20the%20logs.-,NRQL%20syntax%20terms,-If%20you%20need) are wrapped with backticks.
```

File: docs/destinations/analytics/new-relic.mdx [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+- Reserved words are removed. [Reference](https://docs.newrelic.com/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#reserved-words).
```
Comment (ishachopra0401) [inline docs/destinations/analytics/new-relic.mdx:90]:
```suggestion
- [Reserved words](https://docs.newrelic.com/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data/#reserved-words) are removed.
```

File: docs/destinations/analytics/new-relic.mdx [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+## FAQ
+
+### Where can I find the New Relic Account ID?
```
Comment (ishachopra0401) [inline docs/destinations/analytics/new-relic.mdx:94]:
```suggestion
### Where can I find the New Relic account ID?
```

File: docs/destinations/analytics/new-relic.mdx [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+To find the New Relic account ID, log into your [New Relic dashboard](https://one.newrelic.com). 
+
+Depending on your [user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models/), the steps are listed below:
+
+- **New Relic One user model**:  In the top right section of the dashboard, go to your account settings and navigate to **Administration** > **Organization and access** >  **Accounts**. You can find your account ID here.
+- **Original user model**: In the top right section of your New Relic dashboard, go to your account settings and click on **API keys**. Your account ID will be displayed here.
+
+### Where can I find the New Relic Insert Key?
```
Comment (ishachopra0401) [inline docs/destinations/analytics/new-relic.mdx:103]:
```suggestion
### Where can I find the New Relic insert key?
```

File: docs/destinations/analytics/new-relic.mdx [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+
+To find your New Relic insert key, follow these steps:
+
+1. Log into your [New Relic dashboard](https://one.newrelic.com).
+2. Go to **API Keys**.
+3. Find the key of type **INGEST-LICENSE**, as shown:
```
Comment (ishachopra0401) [inline docs/destinations/analytics/new-relic.mdx:109]:
```suggestion
3. Find the key of type **INGEST-LICENSE** (used as insert key), as shown:
```

File: docs/destinations/analytics/new-relic.mdx [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+title: "New Relic"
+description: Step-by-step guide on sending your event data from RudderStack to New Relic.
+---
+
+# New Relic
+
+[New Relic](https://newrelic.com/) is a leading observability platform that lets you monitor, debug, and enhance your tech stack.
+
+RudderStack supports New Relic as a destination to which you can seamlessly send your event data.
+
```
Comment (ishachopra0401) [inline docs/destinations/analytics/new-relic.mdx:11]: <div class="successBlock">

Find the open-source transformer code for this destination in our <a href="https://github.com/rudderlabs/rudder-transformer/tree/master/v0/destinations/new-relic">GitHub repository</a>.
</div>

**Pull request merged**